webpack.config.js

```diff
@@ -3,14 +3,14 @@
  */
 const blocksConfig = require( './tools/webpack/blocks' );
 const developmentConfigs = require( './tools/webpack/development' );
-const interactivity = require( './tools/webpack/interactivity' );
+const scriptModules = require( './tools/webpack/script-modules' );
 const packagesConfig = require( './tools/webpack/packages' );
 const vendorsConfig = require( './tools/webpack/vendors' );
 
 module.exports = [
 	...blocksConfig,
-	interactivity,
 	packagesConfig,
+	scriptModules, // Script modules build depends on the package build.
```

Comment:
Interesting, what is the reason here? I’m curious.

Reply:
I think this is wrong and I'll revert it. I was confusing the build of packages with this "packagesConfig" as I debugged some of the entrypoint issues.